README.md

```diff
@@ -79,13 +72,13 @@ Build the container.  Here, `app-ui` is used as the container name.
 
 ```
 cd app-ui
-docker build --build-arg "NODE_ENV=production" -t app-ui .
```

Comment:
Should NODE_ENV be kept in the readme?

Reply:
In order to use that the Docker needs to be updated. Will do here...